package.json

```diff
@@ -51,38 +66,40 @@
   },
   "homepage": "https://github.com/ERC725Alliance/erc725.js#readme",
   "devDependencies": {
-    "@types/chai": "^4.3.4",
-    "@types/isomorphic-fetch": "^0.0.36",
-    "@types/jest": "^27.5.2",
-    "@types/mocha": "^10.0.1",
-    "@types/node": "^18.11.19",
-    "@types/sinon": "^10.0.13",
-    "@typescript-eslint/eslint-plugin": "^5.50.0",
-    "@typescript-eslint/parser": "^5.50.0",
-    "assert": "^2.0.0",
-    "chai": "^4.3.7",
-    "eslint": "^8.33.0",
+    "@types/chai": "~4.3.11",
+    "@types/jest": "~27.5.2",
+    "@types/mocha": "~10.0.6",
+    "@types/node": "^20.11.16",
+    "@types/sinon": "~10.0.20",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
+    "@typescript-eslint/parser": "^6.21.0",
+    "assert": "^2.1.0",
+    "chai": "~4.4.1",
+    "cross-env": "^7.0.3",
+    "eslint": "^8.56.0",
     "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-prettier": "^8.6.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-prettier": "^5.0.0",
-    "isomorphic-fetch": "^3.0.0",
-    "jest": "^27.5.1",
-    "mocha": "^10.0.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-prettier": "^5.1.3",
+    "esm": "^3.2.25",
+    "jest": "^29.7.0",
+    "mocha": "^10.2.0",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
-    "prettier": "^3.0.0",
-    "sinon": "^15.0.1",
+    "prettier": "^3.2.5",
+    "sinon": "^17.0.1",
     "standard-version": "^9.5.0",
-    "ts-node": "^10.9.1",
-    "typescript": "^4.9.5",
-    "web3": "^1.10.0"
+    "ts-node": "^10.9.2",
+    "tsup": "^8.0.1",
+    "typescript": "^5.3.3",
+    "web3": "^4.4.0"
   },
   "dependencies": {
-    "add": "^2.0.6",
-    "ethereumjs-util": "^7.1.5",
-    "web3-eth-abi": "^1.10.0",
-    "web3-providers-http": "^1.10.0",
-    "web3-utils": "^1.10.0"
+    "cross-fetch": "^4.0.0",
+    "web3-eth-abi": "^4.2.0",
+    "web3-eth-accounts": "^4.1.1",
+    "web3-providers-http": "^4.1.0",
+    "web3-utils": "^4.1.1",
+    "web3-validator": "^2.0.4"
```

Comment:
there is no version 4 for this sub-package of web3?

Reply:
There is a backward compatibility package called "web3" that includes all the other packages. Since we don't require them it's just additional stuff we don't need to pull in. Once you pull in web3 you get everything and the kitchen sink because it's an object with nested objects.

Reply:
I don't think so, because web3-validator is new. I will double check.